diff --git a/lib/lorina/lorina/blif.hpp b/lib/lorina/lorina/blif.hpp
index be49695..1597a63 100644
--- a/lib/lorina/lorina/blif.hpp
+++ b/lib/lorina/lorina/blif.hpp
@@ -228,8 +228,8 @@ public:
 
   virtual void on_latch( const std::string& input, const std::string& output, const std::optional<latch_type>& type, const std::optional<std::string>& control, const std::optional<latch_init_value>& init_value ) const
   {
-    _os << fmt::format( "\n.latch {0} {1} {2} {3} {4}", input, output,
-                        ( type ? *type : latch_type::NONE ), ( control ? *control : "" ), ( init_value ? *init_value : latch_init_value::UNKNOWN ) ) << std::endl;
+    // _os << fmt::format( "\n.latch {0} {1} {2} {3} {4}", input, output,
+                        // ( type ? *type : latch_type::NONE ), ( control ? *control : "" ), ( init_value ? *init_value : latch_init_value::UNKNOWN ) ) << std::endl;
   }
 
   virtual void on_gate( const std::vector<std::string>& inputs, const std::string& output, const output_cover_t& cover ) const
